app/javascript/packs/application.js

```diff
@@ -4,6 +4,7 @@
 // that code so it'll be compiled.
 
 import Rails from '@rails/ujs';
+import '@hotwired/turbo-rails';
```

Comment:
As this one is from external packages, how about place it in a separate group?  this groups import packages are build in rails.

Reply:
Why not, changed ;-)